src/dev/jest/setup/polyfills.js

```diff
@@ -23,17 +23,6 @@ const bluebird = require('bluebird');
 bluebird.Promise.setScheduler(function (fn) { global.setImmediate.call(global, fn); });
 
 const MutationObserver = require('mutation-observer');
-// There's a bug in mutation-observer around the `attributes` option
```

Comment:
1 of 2. There's another polyfill file in x-pack.

https://github.com/elastic/kibana/blob/master/x-pack/dev-tools/jest/setup/polyfills.js#L15-L25

Reply:
Thanks for catching, I've updated x-pack's as well